config_files/config.yaml

```diff
@@ -111,16 +111,19 @@ training:
   main_rank: 0
   train_batch_size: ${data.train_dataloader.config.batch_sampler.config.batch_size}
   global_num_seen_samples: ${modalities_setup.settings.global_num_seen_samples}
+  do_apply_activation_checkpointing: True
```

Comment:
We could think about adding `ActivationCheckpointing` a component. 
See my comment below regarding introducing a `Block` enum.